src/Microsoft.Health.Fhir.Shared.Api/Features/Filters/OperationOutcomeExceptionFilterAttribute.cs

```diff
@@ -81,6 +82,7 @@ public override void OnActionExecuted(ActionExecutedContext context)
                     case ResourceNotValidException _:
                     case BadRequestException _:
                     case RequestNotValidException _:
+                    case BundleEntryLimitExceededException _:
```

Comment:
nit: might be helpful to add a UT to make sure this exception is mapped to BadRequest